test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/TxnUtils.java

```diff
@@ -222,9 +230,30 @@ public static Timestamp defaultTimestampPlusSecs(long offsetSecs) {
 				.setNanos(instant.getNano() - nanosBehind.addAndGet(1)).build();
 	}
 
+	private static int NANOS_IN_A_SECOND = 1_000_000_000;
+	private static AtomicInteger NEXT_NANO = new AtomicInteger(0);
+	private static int NANO_OFFSET = (int) System.currentTimeMillis() % 1_000;
+
+	public static synchronized Timestamp getUniqueTimestampPlusSecs(long offsetSecs) {
+		Instant instant = Instant.now(Clock.systemUTC());
+
+		int candidateNano = NEXT_NANO.getAndIncrement() + NANO_OFFSET;
+		if( candidateNano >= NANOS_IN_A_SECOND ) {
+			candidateNano = 0;
+			NEXT_NANO.set(1);
+		}
+
+		Timestamp uniqueTS = Timestamp.newBuilder()
+				.setSeconds(instant.getEpochSecond() + offsetSecs)
+				.setNanos(candidateNano).build();
+
+		log.info("timestamp : {}", uniqueTS);
+		return uniqueTS;
+	}
```

Comment:
If we are using the first 100 system accounts for the testing, is the adjustment of the timestamp still needed?